packages/store/src/deploymentsSlice/thunks.ts

```diff
@@ -101,7 +129,7 @@ const isWithinFirstMonth = expirationDate => {
   return endOfFirstMonth > new Date();
 };
 
-const trackDeploymentCreation = (totalDeploymentCount, hasDeployments, trial_expiration) => {
+const trackDeploymentCreation = (totalDeploymentCount: number, hasDeployments: boolean, trial_expiration: string | null) => {
```

Comment:
```suggestion
const trackDeploymentCreation = (totalDeploymentCount: number, hasDeployments: boolean, trial_expiration?: string) => {
```
considering the type for `trial_expiration` in `TenantV1`